README.md

```diff
@@ -106,6 +108,7 @@ Usage: test-storybook [options]
 | `-s`, `--stories-json`          | Run in stories json mode (requires a compatible Storybook) <br/>`test-storybook --stories-json`                           |
 | `-c`, `--config-dir [dir-name]` | Directory where to load Storybook configurations from <br/>`test-storybook -c .storybook`                                 |
 | `--watch`                       | Run in watch mode <br/>`test-storybook --watch`                                                                           |
+| `--url`                         | Define browsers to run tests in. One or multiple of: chromium, firefox, webkit <br/>`test-storybook --browsers firefox`   |
```

Comment:
```suggestion
| `--url`                         |  The storybook URL to test against (also configurable via the TARGET_URL environment variable <br/>`test-storybook --url http://the-storybook-url-here.com`   |
```

Reply:
Also is there a way to specify the default value in `commander` and get that in the help string for free?

Reply:
For sure!